lib/components/InputBox/NumberInputBox.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, useState } from "react";
+import styled from "styled-components";
+
+import InputBox, { BaseInputBoxProps } from "./BaseInputBox";
+import Button from "../Button";
+
+const NumberInputBoxWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+`;
+
+const StyledInputBox = styled(InputBox)`
+  margin: 0px;
+
+  text-align: center;
+`;
+
+const StyledButton = styled(Button)`
+  font-size: 24px;
+  font-weight: bold;
+`;
+
+const NumberInputBox: FC<BaseInputBoxProps> = (props) => {
+  let [inputBoxValue, setinputBoxValue] = useState(0);
+
+  const onAdd = () => {
+    inputBoxValue++;
+    setinputBoxValue(inputBoxValue);
+  };
+
+  const onSubtract = () => {
+    if (inputBoxValue > 0) inputBoxValue--;
+    setinputBoxValue(inputBoxValue);
+  };
+
+  return (
+    <NumberInputBoxWrapper>
+      <StyledButton onClick={onSubtract}>-</StyledButton>
```

Comment:
I believe @fdepin intended for us to use icons here, you can use the iconOnly button for that, with `<Button iconOnly>{mdiIconHere}</Button>`

Reply:
done